### PR TITLE
formats!!!

### PR DESCRIPTION
**Pull request by Kqliber**
plugin actually builds now. 
formats are here, with some commands
all has been tested :)

### CODE REVIEW

File: plugin/src/main/java/at/helpch/chatchat/ChatChatPlugin.java [hunk 1]
```diff
@@ -1,20 +1,59 @@
...
+    private BukkitAudiences audiences;
 
     @Override
     public void onEnable() {
+        configManager.reload();
+        audiences = BukkitAudiences.create(this);
+
+        // command registration
+        var commandManager = BukkitCommandManager.create(this);
+        List.of(
```
Comment (iGabyTM) [inline plugin/src/main/java/at/helpch/chatchat/ChatChatPlugin.java:31]: Can be replaced with `Stream#of`

File: plugin/src/main/java/at/helpch/chatchat/ChatChatPlugin.java [hunk 2]
```diff
@@ -1,20 +1,59 @@
...
+                new MainCommand(this),
+                new ReloadCommand(this)
+        ).forEach(commandManager::registerCommand);
+
+        // event listener registration
+        List.of(
```
Comment (iGabyTM) [inline plugin/src/main/java/at/helpch/chatchat/ChatChatPlugin.java:37]: Can be replaced with `Stream#of`

**Review by Starmism — APPROVED** [review]
:shipit:

File: plugin/src/main/java/at/helpch/chatchat/listener/ChatListener.java [hunk 3]
```diff
@@ -1,13 +1,77 @@
...
 
 public final class ChatListener implements Listener {
 
+    private final @NotNull ChatChatPlugin plugin;
+
+    public ChatListener(@NotNull final ChatChatPlugin plugin) {
+        this.plugin = plugin;
+    }
+
     @EventHandler
```
Comment (iGabyTM) [inline plugin/src/main/java/at/helpch/chatchat/listener/ChatListener.java:23]: Add `ignoreCancelled = true` maybe?

**Review by iGabyTM — CHANGES_REQUESTED** [review]

File: plugin/src/main/java/at/helpch/chatchat/ChatChatPlugin.java (hunk 1 above)
Comment (Starmism) [reply plugin/src/main/java/at/helpch/chatchat/ChatChatPlugin.java:31]: Why? No reason to make an extra Stream for no reason. List#of uses the iterator, not a stream.

File: plugin/src/main/java/at/helpch/chatchat/ChatChatPlugin.java (hunk 2 above)
Comment (Starmism) [reply plugin/src/main/java/at/helpch/chatchat/ChatChatPlugin.java:37]: Same as above ^

File: plugin/src/main/java/at/helpch/chatchat/listener/ChatListener.java (hunk 3 above)
Comment (Starmism) [reply plugin/src/main/java/at/helpch/chatchat/listener/ChatListener.java:23]: ➕

**Review by BlitzOffline — APPROVED** [review]

**Review by BlitzOffline — APPROVED** [review]

**Pull request merged**